### ISSUE TITLE
Accessibility (a11y) improvements

### ISSUE THREAD

**Issue by bobbingwide** [bobbingwide/oik-bob-bing-wide#72]
Implement accessibility improvements, primarily for the front end.

### Requirements
- Satisfy WCAG 2.1 AA
- See also https://github.com/bobbingwide/oik/issues/79

### Proposed solution

- Update and PHPUnit test with shared library changes for oik, oik-libs and other plugins which deliver shared libraries.
- Update PHPUnit tests to support PHPUnit 10 & 11.
- Test with WAVE and/or axe Dev Tools

**Comment by bobbingwide** [comment]
Images in the nivo slider do not have an alt attribute, not even an empty one so they each get a **Missing alt text** Error from WAVE
